3.10/data-science.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: default
+description: >- 
+   ArangoDB applies analytics and machine learning to graph data at scale
```

Comment:
YAML indentation should use 2 spaces.
This sounds as if ArangoDB would autonomously process graphs without user interaction. How about this?
```suggestion
  ArangoDB lets you apply analytics and machine learning to graph data at scale
```